frontend/src/constants/roles.js

```diff
@@ -0,0 +1,20 @@
+const ROLES = {
```

Comment:
What's the rationale for not just loading this from the response?

Reply:
Ah, the reason for this is that we're not necessarily using the descriptions from the backend. Such as the credit trade statuses (we have submitted and accepted in the backend, but Sign 1/2 and Sign 2/2 for the front-end).

I was debating if we should just update the description in the roles table. We can use that. But I was thinking there was a different intent originally when they designed the table. So I opt to just do a description in the front-end.